src/native/components/Lotes/CarouselItem.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image, View } from 'react-native';
+
+const CarouselItem = ({
+  obra,
+}) => {
+  const keyExtractor = item => item.id;
+
+  if (obra.images !== undefined && obra.images !== null) {
```

Comment:
We have checked against length on admin, but that was because the `images` array could be returned defined but just blank which I think would also pass thru this condition.

Reply:
so we should check for length here too? or only

Reply:
i was seeing `undefined` in console when logging them